Buildings/Fluid/Dehumidifiers/Desiccant/BaseClasses/PartialDesiccant.mo

```diff
@@ -0,0 +1,264 @@
+        iconTransformation(extent={{90,70},{110,90}})));
+
+protected
+  Buildings.Fluid.Interfaces.PrescribedOutlet outCon(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial)
+    "Model to set outlet conditions"
+    annotation (Placement(transformation(extent={{-6,-110},{14,-90}})));
+  Buildings.Fluid.Dehumidifiers.Desiccant.BaseClasses.Performance dehPer(
+    final vPro_nominal=vPro_nominal,
+    final VPro_flow_nominal=VPro_flow_nominal,
+    final vReg_nominal=vReg_nominal,
+    final VReg_flow_nominal=VReg_flow_nominal,
+    final QReg_flow_nominal=QReg_flow_nominal,
+    final per=perDat)
+    "Calculate the performance of the dehumidifier"
+    annotation (Placement(transformation(extent={{-58,-94},{-38,-74}})));
+  Modelica.Blocks.Sources.RealExpression VPro_flow(
+      final y(final unit="m3/s")= outCon.port_a.m_flow/Medium1.density(
+      state=Medium1.setState_phX(
+      p=port_a1.p,
+      h=port_a1.h_outflow,
+      X=port_a1.Xi_outflow)))
+    "Process air volume flow rate"
+    annotation (Placement(transformation(extent={{-110,-106},{-90,-86}})));
+  Modelica.Blocks.Sources.RealExpression TProEnt(
+    final y(final unit="K")=
+    Medium1.temperature(Medium1.setState_phX(
+    p=port_a1.p,
+    h=inStream(port_a1.h_outflow),
+    X=inStream(port_a1.Xi_outflow))))
+    "Temperature of the process air entering the dehumidifier"
+    annotation (Placement(transformation(extent={{-110,-38},{-90,-18}})));
+  Modelica.Blocks.Sources.RealExpression TRegEnt(
+    final y(final unit="K")=
+    Medium2.temperature(Medium2.setState_phX(
+    p=port_a2.p,
+    h=inStream(port_a2.h_outflow),
+    X=inStream(port_a2.Xi_outflow))))
+    "Temperature of the regeneration air entering the dehumidifier"
+    annotation (Placement(transformation(extent={{-110,-54},{-90,-34}})));
+   Buildings.Fluid.MixingVolumes.MixingVolumeMoistAir
+    vol(redeclare final package Medium = Medium2,
+    massDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    final m_flow_nominal=m2_flow_nominal,
+    final energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    V=m2_flow_nominal*tau/rho_Reg_default,
+    nPorts=1)
+    "Volume for the regeneration air stream"
+     annotation (Placement(transformation(extent={{-105,66},{-85,46}})));
+  Modelica.Blocks.Math.Gain gai1(
+    final k=-1)
+    "Find the opposite number of the input"
+    annotation (Placement(transformation(extent={{0,0},{-20,20}})));
+  Modelica.Blocks.Math.Gain gai2(
+    final k=-1)
+    "Find the opposite number of the input"
+    annotation (Placement(transformation(extent={{0,30},{-20,50}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow preHeaFlo
+    "Prescribed heat flow"
+    annotation (Placement(transformation(extent={{-62,50},{-82,30}})));
+  Modelica.Blocks.Sources.RealExpression mPro_flow(
+   final y(final unit="kg/s")= outCon.port_a.m_flow)
```

Comment:
one more space before `final`. In general, make sure the format looks consistent.

Reply:
Fixed the inconsistent format.